app/src/main/java/protect/card_locker/Group.java

```diff
@@ -1,8 +1,10 @@
 package protect.card_locker;
 
 import android.database.Cursor;
+import android.os.Parcel;
+import android.os.Parcelable;
 
-public class Group
+public class Group implements Parcelable
```

Comment:
Why did you need to make this parcelable?

Reply:
it was made so that ManageGroupsActivity would start ManageGroupActivity by loading a group into an intent